### DISCUSSION TITLE
使用ChatOpenAI模型报如下错误

### DISCUSSION THREAD

**Discussion by huicewang** [FlowiseAI/Flowise#889] [Q&A]
Request failed with status code 404 and body {"error":{"message":"This is not a chat model and thus not supported in the v1/chat/completions endpoint. Did you mean to use v1/completions?","type":"invalid_request_error"

**Comment by chungyau97** [comment]
Close Duplicate

https://github.com/FlowiseAI/Flowise/issues/890
